### PR TITLE
GameDB: Add new entries part 1

### PR DESCRIPTION
**Pull request by RedDevilus**
Partial revisioning and adding new entries, first commit mostly focused on SLES. Others will be checked as well later.

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -1664,8 +1664,8 @@ Name   = Le Monde des Bleus 2003
 Region = PAL-F
 ---------------------------------------------
 Serial = SCES-51041
-Name   = This is Football 2003
-Region = PAL-Unk
+Name   = This is Football 2003
```
Comment (Florin9doi) [inline bin/GameIndex.dbf:1667]:
```
sed -i 's/ $//g' GameIndex.dbf
```

Comment (refractionpcsx2) [reply bin/GameIndex.dbf:1667]: ?

**Comment by RedDevilus** [comment]
Something to keep in mind: Sony in all their wisdom seldom use country code (standard iso-3611-1)
image
But use other region codes from what i found: 

A/AU for Australia (but they seem to prefer E more or put /ANZ in the serial)
AR for Arabic
C for China
CZ for Czech Republic
D for Danish
DU for Dutch
E for Europe
F for French
FI for Finnish
G for Germany
GR for Greek
HU for Hungarian
I for Italian
J for Japanese
N for Norwegian
P for Portuguese
PL for Polish
R for Russian
S for Spanish
SW for Swedish
T for Turkish
K for Korean 

What PCSX2 does is PAL-M3 for 3 languages supported either by an in-game language selector or via the BIOS language selector. It's purely for informational reasons.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -1804,15 +1817,15 @@ VU0KickstartHack = 1 // Fixes Character SPS.
 ---------------------------------------------
 Serial = SCES-51610
 Name   = This is Football 2004 (Red Devils 2004)
-Region = PAL-F-G
+Region = PAL-Du-F // Belgian.
```
Comment (refractionpcsx2) [inline bin/GameIndex.dbf:1820]: Lower down you capitalise DU, which is it? :P

Comment (RedDevilus) [reply bin/GameIndex.dbf:1820]: Guess i missed that one, i capitalised it now 👍

**Review by lightningterror — APPROVED** [review]

**Pull request merged**